Assets/Scripts/ROS Scripts/ROS Connections v2/ROSManager.cs

```diff
@@ -51,6 +55,7 @@ public class ROSSensorConnectionInput
     public int uniqueID = 0;
 
     // Use this for initialization
+    // Create a drone/sensor as per user input
```

Comment:
Nice to use C# documentation style.
```suggestion
    /// <summary> 
    /// Create drones and sensors as defined by user input
    /// </summary>
```